src/Core/Makefile.am

```diff
@@ -7,3 +13,10 @@ SUBDIRS = \
 		FSpot.Query \
 		FSpot.Gui \
 		FSpot.Platform
+
+RESOURCES = \
+		FSpot.Core.UnitTest/TestData/taglib-sample.xmp \
+		FSpot.Core.UnitTest/TestData/taglib-sample-broken.xmp \
+		FSpot.Core.UnitTest/TestData/taglib-sample.jpg \
+		FSpot.Core.UnitTest/TestData/taglib-sample-broken.jpg
+		
```

Comment:
Debian certainly expects source packages to include test suites [0], and
runs them as part of continuous integration [1]. I didn't set this up in
the .deb I made, but that was because I ran out of time (it took me several
days to get this far!) I'd expect to do it if I was preparing an official
Debian package though.

[0]
http://anonscm.debian.org/gitweb/?p=autopkgtest/autopkgtest.git;a=blob_plain;f=doc/README.package-tests.rst;hb=HEAD
[1] https://ci.debian.net/

On 16 June 2015 at 07:44, Stephen Shaw <notifications@github.com> wrote:

> In src/Core/Makefile.am
> <https://github.com/mono/f-spot/pull/29#discussion_r32494690>:
>
> > @@ -7,3 +13,10 @@ SUBDIRS = \
> >  		FSpot.Query \
> >  		FSpot.Gui \
> >  		FSpot.Platform
> > +
> > +RESOURCES = \
> > +		FSpot.Core.UnitTest/TestData/taglib-sample.xmp \
> > +		FSpot.Core.UnitTest/TestData/taglib-sample-broken.xmp \
> > +		FSpot.Core.UnitTest/TestData/taglib-sample.jpg \
> > +		FSpot.Core.UnitTest/TestData/taglib-sample-broken.jpg
> > +		
>
> Sadly my Automake foo is pretty weak these days, but do we really want the
> distribution tarball to be able to run the unit tests? Or should the unit
> tests just be reserved for local development and testing?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mono/f-spot/pull/29/files#r32494690>.
>